roles/openshift_service_catalog/tasks/generate_certs.yml

```diff
@@ -35,6 +45,15 @@
     signer_key: "{{ generated_certs_dir }}/ca.key"
     signer_serial: "{{ generated_certs_dir }}/apiserver.serial.txt"
 
+- name: Generating Controller Manager keys
```

Comment:
Why aren't we using service serving certs here?  That's what every component should be using to listen on TLS.

Reply:
@smarterclayton I'm not certain why this was done this way originally, I followed what was in place for the catalog API Server.  I can create a new item to cut both of these over to use service serving certs in the next release, or must this be changed for 3.10?